src/app/components/header/header.component.spec.ts

```diff
@@ -1,14 +1,33 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
+import { AuthService } from './../../services/auth.service';
+import {AppSettings} from '../../app.settings';
+import { Router } from '@angular/router';
 
 import { HeaderComponent } from './header.component';
 
+class MockAuthService{
+  isLoggedIn() {
+    return true;
```

Comment:
Yeah, it would be cool to put these in their own files because they'll likely be reused